inference-engine/src/mkldnn_plugin/mkldnn_graph.cpp

```diff
@@ -809,7 +809,27 @@ void MKLDNNGraph::PullOutputData(BlobMap &out) {
             MB_to_process = std::min<int>(config.batchLimit, MB_to_process);
         size_t size_to_copy = intr_blob.GetElementsCount() * MB_to_process / MB;
 
-        cpu_convert(intr_blob_ptr, ext_blob_ptr, srcPrec, dstPrec, size_to_copy);
+        const auto actualDesc = node->getParentEdgeAt(0)->getDesc();
+        const auto expectedDesc = ext_blob->getTensorDesc();
+
+        bool isScalarOutput = false;
```

Comment:
Not sure I like the solution. Need to be discussed offline.

Reply:
comment added